.pre-commit-config.yaml

```diff
@@ -33,13 +33,3 @@ repos:
   rev: v0.942
   hooks:
     - id: mypy
-- repo: https://github.com/nbQA-dev/nbQA
```

Comment:
reverse (maybe your branch is not at HEAD?)

Reply:
It is at HEAD. We forgot to remove those in the PR that converted all notebooks to markdown.

Reply:
Oh right we dont do notebook any more haha